package.json

```diff
@@ -37,7 +37,6 @@
     "body-parser": "^1.19.0",
     "bunyan": "^1.8.15",
     "bunyan-format": "^0.2.1",
-    "bunyan-request-logger": "^2.1.0",
```

Comment:
This module has some serious security probs now, and no fixed version. Think we can do without it as we can always use requests rather than traces (requests gives less info tho :( )